src/test/openads/domain/service/helper/DomainEventBusWrapper.js

```diff
@@ -0,0 +1,11 @@
+import DomainEventBus from '../../../../../openads/domain/service/DomainEventBus'
+
+export default class DomainEventBusWrapper {
```

Comment:
![](https://media.giphy.com/media/Y0AzCrZBulRbG/giphy.gif)

Reply:
The intention of this wrapper was to test a case when calling DomainEventBus static methods and calling them again using an instance of a class which also is calling DomainEventBus using static methods, the expected behavior fulfills. Another way of doing this was importing DomainEventBus twice using different names but lint was complaining about several imports of the same class, so I went for this wrapper solution afterward.

Reply:
The broken link rule is: import/no-duplicates